pkg/libvmi/replicaset.go

```diff
@@ -26,10 +26,11 @@ import (
 )
 
 func NewReplicaSet(vmi *v1.VirtualMachineInstance, replicas int) *v1.VirtualMachineInstanceReplicaSet {
+	const taillength = 5
```

Comment:
Please squash 66e2f8174453cc0aeb00260f8d2f9f2c3f81426e

Reply:
May I ask why you find it important? The first commit just move the code. I prefer to make it easier on the reviewer by highlighting what are my changes and the reasons behind them.

Reply:
You introduce the `NewReplicaSet` without the constant and then fix it. To me it is to some extend as to introduce bug and fix in the same PR. Redundant.

Reply:
There are now two commits which seem good to me.
The second one explains well why something was renamed.

I guess the original comment was already resolved.